Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,22 @@
+
+const fs = require('fs');
+
+fs.readFile('/root/CYF/Module-Data-Groups/Sprint-1/stretch/aoc-2018-day1/input.txt', (err, data) => {
```

Comment:
With the file path specified as an absolute path, the script probably cannot find `input.txt` when it is executed on a different computer.

Assuming `input.txt` will always be in the same folder as the script, how would you modify your code to read data from this `input.txt` file?